### ISSUE TITLE
Implement aiohttp.web executable helper

### ISSUE THREAD

**Issue by asvetlov** [aio-libs/aiohttp#733]
`python -m aiohttp.web HOST PORT pkg.module.main [ARGS]`

See #690 for details

**Comment by asvetlov** [comment]
Questions: 
1. How to support SSL context, do we need it at all?
2. How to skip `HOST` `PORT` parameters and use `localhost 8080`?
3. What `pkg.module.main` should be? I think it is either a regular function or await-able object (a coroutine or a Future) accepting `ARGS`. BTW I suggest `pkg.module:main` syntax to split import path and name inside imported module.
4. Do we want support path-to-file instead of import-path-to-module?
5. What's `[ARGS]` syntax? Is it just `str` params split by spaces or we want to use named parameters and apply type checking/conversions to ones?

**Comment by jashandeep-sohi** [comment]
1. Leave SSL out for now -- could always add a `--ssl` option later on that could create a `SSLContext` from a cert file.
2. `argparse.ArgumentParser` supports optional positional arguments. Might have to move `pkg.module.main` argument first though. `ArgumentParser.parse_known_args` might also be of use to avoid parsing `[ARGS]`.
3. An async function that returns an `web.Application` instance.
4. ?
5. Use the leftover list of args returned by `ArgumentParser.parse_known_args`. The user can take care of the parsing in their `main(args)` implementation, if they need to.

**Comment by rutsky** [comment]
1. Perhaps do `HOST` `PORT` optional keyword arguments?

```
python -m aiohttp.web [--host HOST] [--port PORT] pkg.module.main [ARGS]
```

**Comment by asvetlov** [comment]
Looks good.
Any volunteer?
The PR should use already merged `web.run_app()`.

**Comment by jashandeep-sohi** [comment]
I'll volunteer.

**Comment by asvetlov** [comment]
@jashandeep-sohi thank you too much. It will be significant addition, especially for newcomers.

I want to make aiohttp 0.21 release close to the end of January.
If you will not end the PR up to the date -- that's also fine, no rush. Looks like we have new release every month, 0.22 is expected in between of February last week end and March.

But, please, document the issue in PR. 
You have much fluent English than me :)

**Comment by asvetlov** [comment]
Fixed by #740
